vendor/composer/autoload_classmap.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
hum, isn't a good idea to add vendor/* in repository. You have a composer.json & a composer.lock.

Reply:
ok, but I was not sure will the autoload work in that case? since we are in a module I'm not an expert of multiple composer.json files in the same project

Reply:
All packages will be added automatically when building this module (in addons).